apps/rehabstod/src/components/ExpandableLink/ExpandableLink.tsx

```diff
@@ -0,0 +1,15 @@
+import { IDSIcon } from '@frontend/ids-react-ts'
+import { ReactNode, useState } from 'react'
+
+export function ExpandableLink({ children, title }: { children: ReactNode; title: string }) {
```

Comment:
Should we add some tests for this component?

Reply:
Added tests